tests/queries/tests.py

```diff
@@ -3825,3 +3825,17 @@ def test_ticket_23622(self):
             Ticket23605A.objects.filter(qx),
             [a2], lambda x: x
         )
+
+class Ticket26983Tests(TestCase):
```

Comment:
Not sure, clearly the issue isn't `to_field` specific. Since the fix involves lookups my instinct was to look at `tests/lookup` or another test app that's lookup related.  I think of "queries" as anything that doesn't fit in a more specific app, but if it's easier to keep it in here, I guess it's okay.